public/modules/custom/grants_metadata/src/TypedData/Definition/KuvaToimintaDefinition.php

```diff
@@ -46,6 +46,16 @@ public function getPropertyDefinitions(): array {
           'compensationInfo',
           'generalInfoArray',
           'yearsForMultiYearApplication',
+        ])
+        ->setSetting('relations', [
+          'slave' => 'kyseessa_on_monivuotinen_avustus',
+          'master' => 'tulevat_vuodet_joiden_ajalle_monivuotista_avustusta_on_haettu_ta',
+          'type' => 'truthy',
+        ])
+        ->setSetting('webformDataExtracter', [
```

Comment:
Hmm. Pitäsköhän tää tehä samalla tavalla ku noissa valuecallbackeissa, että konffis tuolla definitionissa arguments ja sit passais ne eteenpäin sille funktiolle?

Meinaan, että ois yleiskäyttösempi ehkä jos jossain tarvitaan jatkossa jotain toista tapaa jumppailla näitä?

Like so:

`      $info['hanke_alkaa'] = DataDefinition::create('string')
        ->setLabel('Hanke alkaa.')
        ->setSetting('jsonPath', [
          'compensation',
          'activityInfo',
          'plannedActivityInfoArray',
          'projectStartDate',
        ])
        ->setSetting('typeOverride', [
          'dataType' => 'string',
          'jsonType' => 'datetime',
        ])
        ->setSetting('valueCallback', [
          'service' => 'grants_metadata.converter',
          'method' => 'convertDates',
          'arguments' => [
            'dateFormat' => 'c',
          ],
        ]);`